app/src/webkit/java/org/mozilla/focus/webkit/WebkitView.java

```diff
@@ -78,10 +82,18 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
     }
 
     @Override
-    public void restoreWebviewState(Bundle savedInstanceState) {
-        // We need to have a different method name because restoreState() returns
-        // a WebBackForwardList, and we can't overload with different return types:
-        final WebBackForwardList backForwardList = restoreState(savedInstanceState);
+    public void restoreWebViewState(Bundle inBundle) {
```

Comment:
\[PMD\] INFO: The method 'restoreWebViewState' has a Cyclomatic Complexity of 5\.